src/databaseTest/java/net/sf/jabref/shared/DBMSConnectionTest.java

```diff
@@ -23,13 +25,13 @@
     }
 
     @Test
-    public void testGetConnection() throws SQLException {
+    public void testGetConnection() throws SQLException, InvalidDBMSConnectionPropertiesException {
```

Comment:
(For the next time:) Just use the generic `throws Exception` in tests so that you don't have to edit tests if you decide to throw a different exception.